platform/ui/src/components/Button/Button.tsx

```diff
@@ -176,6 +233,14 @@ Button.propTypes = {
     'inherit',
     'light',
   ]),
+  border: PropTypes.oneOf([
```

Comment:
From current architecture stand point this design is very hard to maintain and scale.


I believe border props should have something like this
(from borderClasses)
none: '',
  light: 'border border-primary-light',
  primary: 'border border-primary-main',
  primaryActive: 'border border-primary-active',
  secondary: 'border border-secondary-light',
  white: 'border border-white',
  black: 'border border-black',
  
  
  also I have realized color is not consistent

Reply:
we definetly one day should revisit how components has being implemented/design that decrease a lot the style customization